pkgs/applications/virtualization/colima/default.nix

```diff
@@ -42,7 +43,10 @@ buildGoModule rec {
     -X github.com/abiosoft/colima/config.revision=$(cat .git-revision)"
   '';
 
-  subPackages = [ "cmd/colima" ];
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace util/util.go \
+      --replace 'sw_vers' "${darwin.DarwinTools}/bin/sw_vers"
+  '';
```

Comment:
Could we just put `darwin.DarwinTools` in `nativeBuildInputs` instead of this?

Reply:
Yes, indeed. Thank you for the tip.